python/setup.py

```diff
@@ -66,7 +66,7 @@
     os.path.join(pdo_root_dir, 'common/packages/base64')
 ] + openssl_include_dirs
 
-libraries = ['updo-common'] + openssl_libs
+libraries = ['updo-common'] + ['updo-crypto'] + openssl_libs
 
```

Comment:
why are we doing multiple list concatenations? ['updo-common' 'updo-crypto'] + openssl_libs

Reply:
no reason, will update

Reply:
done!